include/cute/arch/copy_xe_U4.hpp

```diff
@@ -36,6 +36,22 @@
 
 namespace cute
 {
+struct XE_2D_U4x8x16_LD_T {
```

Comment:
Can you add tests for the new copy atoms in /test/unit/cute/intel_xe/copy_block.cpp?